genesis/genesis_kopernikus.json

```diff
@@ -117,7 +117,7 @@
           {
             "amount": 4000000000000,
             "nodeID": "NodeID-AK7sPBsZM9rQwse23aLhEEBPHZD5gkLrL",
-            "validatorDuration": 31536000,
+            "validatorDuration": 63072000,
```

Comment:
I suggest to not mix topics - this is some separate feature request (??)

Reply:
Yes, had to do it for local testing, otherwise kopernikus genesis is not working
I can create a small pull-request with genesis changes preceding this one, wdyt?

Reply:
https://github.com/chain4travel/caminogo/pull/292